link_shortener/templates/base.html

```diff
@@ -71,6 +71,46 @@
     {% block content %} {% endblock %}
   </body>
   <script>
+    // getting unique owners
+    function addOwnersToSelection() {
+      let ownerarr = [];
+      let owners = document.getElementsByClassName("rowowner");
+      let ownerfilter = document.getElementById("ownerfilter");
+      for (i = 0; i < owners.length; i++) {
+        ownerarr.push(owners[i].innerHTML);
+      }
```

Comment:
I wonder what you'd think about this alternative? You don't need to implement if you don't like it, but we can discuss it with Alex / Kája at our next progress check.

```
Array.prototype.forEach.call(owners, (owner) => {
  ownerarr.push(owner.innerHTML);
})
```